### PR TITLE
Build updates

### PR DESCRIPTION
**Pull request by devnill**
Removed gulp depends and set up webpack to make the FE build.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +29,7 @@ A volunteer database for events written using the Laravel 5.1 framework
 4. Optionally, configure your queue and broadcast drivers. If you want to use websockets, you'll need to use redis for broadcasting
 5. In the **laravel/resources/js/** folder, copy **config.example.js** and rename it to **config.js**
 6. Optionally, you may configure your websocket server to use a specific hostname, however by default it will use the current domain of the site
-7. Run ```gulp``` within the **laravel** folder
+7. Run ```./node_modules/.bin/webpack``` within the **laravel** folder.
```
Comment (itsrachelfish) [inline README.md]: This is stinky :hankey: ... should be a symlink in the root of the project or something

File: laravel/package.json [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+    "babel-core": "^6.22.1",
+    "babel-loader": "^6.2.10",
+    "babel-preset-es2015": "^6.22.0",
+    "css-loader": "^0.26.1",
+    "dotenv": "^4.0.0",
+    "node-sass": "^4.5.0",
+    "path": "^0.12.7",
+    "sass-loader": "^4.1.1",
+    "style-loader": "^0.13.1",
+    "webpack": "^2.2.1"
```
Comment (itsrachelfish) [inline laravel/package.json]: I don't like how many more dependencies this is...

File: laravel/resources/js/base.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
+require('../css/main.scss');
```
Comment (itsrachelfish) [inline laravel/resources/js/base.js]: Shouldn't the main build file go in the parent directory if it's going to be loading both css and javascript?

File: README.md (hunk 1 above)
Comment (devnill) [inline README.md]: Adding ```npm build``` to package.json

File: laravel/package.json (hunk 2 above)
Comment (devnill) [inline laravel/package.json]: webpack is very granular -_-

**Pull request merged**